mitosheet/mitosheet/tests/steps_performers/test_graph.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+import pandas as pd
+from mitosheet.tests.test_utils import create_mito_wrapper_dfs
+
+def test_create_graph():
+    df = pd.DataFrame({'A': ['aaron', 'jake', 'nate'], 'B': [1, 2, 3]})
+    mito = create_mito_wrapper_dfs(df)
+    mito.generate_graph(
+        {
+            'safety_filter_turned_on_by_user': False
+        },
+        {
+            'graph_type': 'bar',
+            'sheet_index': 0,
+            'x_axis_column_ids': ['A'],
+            'y_axis_column_ids': ['B']
+        },
+        {},
+        {
+            'height': 400,
+            'width': 400
+        }
+    )
+    
+    assert len(mito.steps) == 2
+    assert mito.curr_step.step_type == 'graph'
+
+
```

Comment:
Now that we have some persistent data, can you test this? I'm looking to check that the column IDs get saved properly, that the javascript, etc are defined, etc. 

Just make sure it actually saves what you expect it to!

If you can create an accessor in the Mito Test Helper thing, so that refactoring internal state makes updating tests easy, that would be great!